### PR TITLE
Fix admin user office

### PR DESCRIPTION
**Pull request by reinbach**
If schema is not selected in admin, user admin interface errors out as it is unable to find `UserTenantProfile`

Add migration for `reports.Office` sequence ids needed to be reset due to old ids being copied across

Correct reports `UserTenantProfile ` data migration

### CODE REVIEW

File: src/etools/applications/reports/migrations/0023_auto_20191014_1546.py [hunk 1]
```diff
@@ -1,27 +1,33 @@
...
     for each country that UserProfile has available
     Use the connection to determine the current country
     """
-    UserProfile = apps.get_model("users", "userprofile")
-    UserTenantProfile = apps.get_model("reports", "usertenantprofile")
-    Office = apps.get_model("reports", "office")
-    country = connection.tenant
-    country_name= getattr(country, "name", None)
-    if country_name and country_name != get_public_schema_name():
+    if connection.tenant.schema_name != "test":
```
Comment (domdinicola) [inline src/etools/applications/reports/migrations/0023_auto_20191014_1546.py]: test or public?

File: src/etools/applications/reports/migrations/0023_auto_20191014_1546.py [hunk 2]
```diff
@@ -1,27 +1,33 @@
...
-            if country in profile.countries_available.all():
-                office = Office.objects.get(pk=profile.office.pk)
-                UserTenantProfile.objects.create(
-                    profile=profile,
-                    office=office,
-                )
+            if profile.office and country in profile.countries_available.all():
+                try:
+                    office = Office.objects.get(pk=profile.office.pk)
+                except Office.DoesNotExist:
```
Comment (domdinicola) [inline src/etools/applications/reports/migrations/0023_auto_20191014_1546.py]: I think we should not catch this... we have to find these issues

File: src/etools/applications/reports/migrations/0024_auto_20191122_1559.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.2.7 on 2019-11-22 15:59
+
+from django.db import connection, migrations
+from django.db.models import Max
+
+
+def reset_office_seq(apps, schema_editor):
+    Office = apps.get_model("reports", "office")
```
Comment (domdinicola) [inline src/etools/applications/reports/migrations/0024_auto_20191122_1559.py]: we can add here the `if connection.tenant.schema_name not in ["test", "public"]

File: src/etools/applications/reports/migrations/0023_auto_20191014_1546.py (hunk 1 above)
Comment (domdinicola) [reply src/etools/applications/reports/migrations/0023_auto_20191014_1546.py]: `if connection.tenant.schema_name not in ["test", "public"]

File: src/etools/applications/users/admin.py [hunk 4]
```diff
@@ -18,6 +19,15 @@
 from etools.libraries.azure_graph_api.tasks import sync_user
 
 
+def get_office(obj):
```
Comment (domdinicola) [inline src/etools/applications/users/admin.py]: :100:

File: src/etools/applications/reports/migrations/0023_auto_20191014_1546.py (hunk 2 above)
Comment (reinbach) [reply src/etools/applications/reports/migrations/0023_auto_20191014_1546.py]: We don't actually want to catch these as the offices for the user may be in other schemas, and not available in the current schema

File: src/etools/applications/reports/migrations/0024_auto_20191122_1559.py (hunk 3 above)
Comment (reinbach) [reply src/etools/applications/reports/migrations/0024_auto_20191122_1559.py]: Done

File: src/etools/applications/reports/migrations/0023_auto_20191014_1546.py (hunk 1 above)
Comment (reinbach) [reply src/etools/applications/reports/migrations/0023_auto_20191014_1546.py]: Updated it to both

**Pull request merged**